doc/PROTOCOL-WEB.md

```diff
@@ -48,8 +48,14 @@ Content-Type
 HTTP wire protocols
 
 1. support any HTTP/*, with no dependency on HTTP/2 specific framing
-2. use lower-case header/trailer names
```

Comment:
How about 

2. header names may not be lower-case over HTTP/1.1; and trailers are not supported

Reply:
I believe the current language is referring to the block of trailers encoded in the last length-prefixed message in the stream, not HTTP trailers. Those trailers aren't visible to any generic HTTP systems (proxies, middleware, etc.) so I think it's fine to mandate lower-case field names on the wire.

How about this?

> HTTP header and trailer names may not be lower-case over HTTP/1.1; trailers encoded in the final length-prefixed message of the stream must use lower-case names

Reply:
@wenbozhu Could you take a look and approve the PR if the updated text looks good? Thanks!